bucket/alacritty.json

```
@@ -6,18 +6,47 @@
     "suggest": {
         "vcredist": "extras/vcredist2017"
     },
-    "url": "https://github.com/alacritty/alacritty/releases/download/v0.10.0/Alacritty-v0.10.0-portable.exe#/alacritty.exe",
-    "hash": "6253cc8b39c784f0369ab6e307a49c473eb81863df4ec7882ecfb6d7630a8811",
-    "bin": "alacritty.exe",
+    "url": [
+        "https://github.com/alacritty/alacritty/releases/download/v0.10.0/Alacritty-v0.10.0-portable.exe#/alacritty.exe",
+        "https://github.com/alacritty/alacritty/releases/download/v0.10.0/alacritty.yml"
+    ],
+    "hash": [
+        "6253cc8b39c784f0369ab6e307a49c473eb81863df4ec7882ecfb6d7630a8811",
+        "c7c2f96113f3c2cdf0ef059cded558cc5746b3be60c6fef3b7040a3b8325c686"
+    ],
+    "bin": [
+        [
+            "alacritty.exe",
+            "alacritty.exe",
+            "--working-directory \"%USERPROFILE%\""
+        ],
```

```suggestion
        "alacritty.exe",
```

This flag wasn't present before, so we should not add it.